tox.ini

```diff
@@ -114,8 +115,8 @@ extras =
     alldeps: test_all
 
 install_command =
-    !devdeps: python -I -m pip install
-    devdeps: python -I -m pip install -v --pre
+    !devdeps: python -I -m pip install --prefer-binary
+    devdeps: python -I -m pip install -v --pre --prefer-binary
```

Comment:
note to reviewers: this change is not related to the issue with `attrs`, but emerged from the discussion with mpl maintainer Thomas Caswell; it is intended to protect CI against situations like the one we saw yesterday on https://github.com/matplotlib/matplotlib/issues/28551

Reply:
I also vaguely remember a time where "prefer binary" bit us but I don't remember why, so why not.